src/main/java/org/skife/jdbi/v2/SQLStatement.java

```diff
@@ -245,11 +245,6 @@ protected String getSql()
         return sql;
     }
 
-    protected Binding getParameters()
-    {
-        return params;
-    }
-
```

Comment:
Strictly speaking this is a backward-incompatible change.

From a practical standpoint though, `SQLStatement` isn't intended to be extended by clients.